img/c/csas.cz.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="37" height="37"><path d="M18.502.075c1.276 0 2.394.476 3.322 1.424.931.943 1.404 2.094 1.404 3.396 0 1.332-.464 2.468-1.394 3.412-.93.947-2.057 1.421-3.333 1.421-1.301 0-2.423-.474-3.334-1.421-.927-.943-1.392-2.08-1.392-3.412 0-1.301.465-2.441 1.378-3.383C16.079.561 17.2.075 18.502.075zm15.692 19.117V16.26a5.026 5.026 0 00-5.022-5.024H7.828a5.026 5.026 0 00-5.02 5.024V27.3h23.812v1.675H2.806v2.929a5.021 5.021 0 005.021 5.021h21.345a5.022 5.022 0 005.021-5.021V20.865H10.386v-1.673z" fill="#e30613"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" data-name="Ebene 2" viewBox="0 0 229 229"><rect width="229" height="229" fill="#276be3" fill-rule="evenodd" rx="425.8" ry="229"/><path fill="#fff" d="M56.8 154.9h88.4V143H56.9c0-1.4 0-28.1-.2-40.5-.1-10.6 7-20.8 20.7-20.6 24.8.4 49.6.4 74.4 0a19.7 19.7 0 0 1 20.4 20.4v6.3H83.7v11.6h88.5V161c0 12.6-8 20.5-20.8 20.5H77.6c-12.8 0-20.8-8-20.8-20.8v-5.8m57.8-85.4a19 19 0 0 1 0-38 19 19 0 0 1 0 38" data-name="Ebene 1"/></svg>
```

Comment:
Why not just link to [this image](https://github.com/2factorauth/twofactorauth/pull/7785/files#diff-c220e6045f27560a11f3e02dfc00148db6cc6c011c889c2f70c501dab9926c94)?

Reply:
For entries that are more or less loosely connected like these, we'd have to be careful in the future not to update both if only one should have been changed. With entries like _Santander_ or _Vodafone_ it's a no-brainer - but here?